setup.py

```diff
@@ -131,6 +131,14 @@
     long_description_content_type="text/markdown",
     url="https://github.com/ag2ai/ag2",
     packages=setuptools.find_packages(include=["autogen*"], exclude=["test"]),
+    package_data={
+        "autogen.agentchat.contrib.captainagent": [
+            "tools/**/*.py",  # CaptainAgent tools
+            "tools/tool_description.tsv",
+            "tools/requirements.txt",
+        ]
```

Comment:
Comparing with https://github.com/microsoft/FLAML/blob/b9194cdcf216a43d0f686cf548873d7bf6829e20/setup.py#L33
I wonder:
```suggestion
        "autogen.agentchat.contrib.captainagent.tools": [
            "tool_description.tsv",
            "requirements.txt",
        ]
```

Reply:
Thanks for the chat regarding including `__init__.py`, @sonichi.

I've now tested a number of scenarios and to include the tools files with descriptions and requirements, the following works without warnings.

```
    packages=setuptools.find_namespace_packages(
        include=[
            "autogen*",
            "autogen.agentchat.contrib.captainagent.tools*",
        ],
        exclude=["test"],
    ),
    package_data={
        "autogen.agentchat.contrib.captainagent": [
            "tools/tool_description.tsv",
            "tools/requirements.txt",
        ]
    },
    include_package_data=True,
```

Testing locally with `pip install -v ".[captainagent]"` I can run a CaptainAgent notebook file.